### ISSUE TITLE
How to use for testing or evaluation purpose?

### ISSUE THREAD

**Issue by arunjung1991** [CVI-SZU/UniFace#9]
if not self.training:
            a, b = feat.chunk(2)
            return a + b
        else:
            return self.loss(feat, t, partial_index)
how can it be used for testing or validation dataset?

**Comment by supertx** [comment]
have u handle with this problem.I write some code to test for facial recognization but seem not work.
def compare(f1, f2):
    return torch.cosine_similarity(f1, f2)


def main():
    net = torch.load("/home/xxx/uniface/Models-LResNet50EIR/LResNet50EIR_28th_epoch.pth")
    net.to(torch.device("cpu"))
    net.eval()
    net.training = False
    img1 = openImg("xxx.png")
    img2 = openImg("xxx.png")
    re1 = net(img1.reshape(1, 3, 112, 112))
    re2 = net(img2.reshape(1, 3, 112, 112))
    result = compare(re1, re2)
    return result
